CMakeLists.txt

```diff
@@ -54,7 +54,8 @@ if(FLAGGEMS_USE_EXTERNAL_TRITON_JIT)
 else()
   set(TRITON_JIT_INSTALL ON) # install triton jit
   FetchContent_Declare(TritonJIT
-    GIT_REPOSITORY https://github.com/iclementine/libtorch_example.git
+    GIT_REPOSITORY git@github.com:AdvancedCompiler/libtriton_jit.git
+    # GIT_REPOSITORY https://github.com/iclementine/libtorch_example.git
```

Comment:
I suggest using the original repository. If any code changes are made, please create a pull request to libtorch_example.